src/main/java/pl/wrapper/parking/exception/DummyService.java

```diff
@@ -7,7 +7,6 @@
 
 import static pl.wrapper.parking.result.FailureResult.failureResult;
 
-
```

Comment:
The if check in dummyGetParkingOccupancyByParkingId() is not needed. In either case (is success or not) we still return the very same SuccessResult or FailureResult, just needlessly repackaging its contents into a new object